bin/setup

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+set -vx
+
+bundle install
+
+if [[ ! -d tmp/tree-sitter-math ]]; then
+  mkdir -p tmp
+  git -C tmp/ clone --depth=1 https://github.com/DerekStride/tree-sitter-math.git
+fi
+
+cd tmp/tree-sitter-math
+
+mkdir -p target
+gcc -shared -o target/parser.so -fPIC src/parser.c -I./src
+
+cd ../..
+mkdir -p tree-sitter-parsers
+cp tmp/tree-sitter-math/target/parser.so tree-sitter-parsers/math.so
+
+bundle exec rake compile -- --disable-sys-libs
```

Comment:
Why are you disabling sys lib for the math parser? Wouldn't it be better if it was instructed by the build requirement?

My only concern for the time being is that we're not going to be able to test it with the different options.

Besides, I don't think the flag is stable as of yet. My latest tests with parsers generated from [here](https://github.com/Faveod/tree-sitter-parsers/) failed spectacularly and I haven't had the time to zoom in on it. (I was testing on my local docker so I don't have logs to share).

Reply:
We can remove it, it was just me testing it & I forgot to remove it.